sql/2021/ecommerce/android_ios_app_links_ecomm_sites.sql

```diff
@@ -0,0 +1,36 @@
+#standardSQL
+# This query uses custom metric '_well-known' - https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/well-known.js
+# Note that in this query, there is a subtle bug where the site could have empty /.well-known/assetlinks.json or /.well-known/apple-app-site-association files which will lead to over counting sites with native app links
+# an example is: https://www.allbirds.com/.well-known/assetlinks.json which has a payload of "[]"
+# To fix this, this would require response body parsing on well-known.js
+
+SELECT
+  client,
+  COUNTIF(android_app_links) AS android_app_links,
+  COUNTIF(ios_universal_links) AS ios_universal_links,
+  COUNT(0) AS total,
+  COUNTIF(android_app_links) / COUNT(0) AS pct_android_app_links,
+  COUNTIF(ios_universal_links) / COUNT(0) AS pct_ios_universal_links
+FROM (
+  SELECT DISTINCT
+    _TABLE_SUFFIX AS client,
+    url
+  FROM
+    `httparchive.technologies.2021_07_01_*`
+  WHERE
+    category = 'Ecommerce')
```

Comment:
In other queries you sometimes exclude specific apps. It might be useful to declare a temporary function like `isEcommerce(category, app)` that encapsulates the logic in a reusable way to avoid discrepancies like this.

Reply:
Thanks. Do you think it makes sense to fix it or is it a nice to have?

Reply:
At a minimum this should be updated to include the other apps for consistency with related queries. The temp function is just a nice to have and not necessary for this PR.

Reply:
@rviscomi  and maybe @rockeynebhwani as well - some of the queries from 2020 had category = "Ecommerce" only whereas some had (category =   'Ecommerce' AND app != 'Cart Functionality' AND app != 'Google Analytics Enhanced eCommerce').

I can fix them all to be consistent & re-query the data.. is there any case where the above won't be true and I should look at "Ecommerce" only?